utilities/test_suite/rpp_test_suite_common.h

```diff
@@ -1095,40 +1137,34 @@ inline void compare_reduction_output(Rpp64u* output, string funcName, RpptDescPt
             func += "Tensor_PLN1";
     }
 
-    refFile = scriptPath + "/../REFERENCE_OUTPUT/" + funcName + "/"+ func + ".csv";
+    refFile = scriptPath + "/../REFERENCE_OUTPUT/" + funcName + "/"+ binFile + ".bin";
+
 
-    ifstream file(refFile);
-    Rpp64u *refOutput;
-    refOutput = (Rpp64u *)calloc(srcDescPtr->n * 4, sizeof(Rpp64u));
     string line,word;
     int index = 0;
 
-    // Load the refennce output values from files and store in vector
-    if(file.is_open())
-    {
-        while(getline(file, line))
-        {
-            stringstream str(line);
-            while(getline(str, word, ','))
-            {
-                refOutput[index] = stoi(word);
-                index++;
-            }
-        }
-    }
-    else
-    {
-        cout<<"Could not open the reference output. Please check the path specified\n";
-        return;
-    }
+    FILE *fp;
+    fp = fopen(refFile.c_str(), "rb");
```

Comment:
Try adding a small helper function for reading the data from binary file, instead of readding same code

Reply:
Done